packages/avnu_provider/dart_test.yaml

```
@@ -0,0 +1,3 @@
+tags:
+  integration:
+  unit:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)